src/app/content/highlights/hooks/locationChange.ts

```diff
@@ -26,6 +28,19 @@ const hookBody = (services: MiddlewareAPI & AppServices) => async() => {
   if (highlights.data) {
     dispatch(receiveHighlights(highlights.data));
   }
+
+  if (Object.keys(totalCountsInState).length > 0) { return; }
```

Comment:
what if the data was previously fetched and the user just has no highlights? can we initialize it to null and check for that instead?

Reply:
I've update this, but it is still not solving issue that this hook does not have last version of state which causes summary request at each location change even that totalCountsPerPage are in state